src/Bundle/StorageBundle/Resources/config/services.yml

```diff
@@ -28,4 +28,9 @@ services:
     UnitedCMS\StorageBundle\Command\StorageFileDeleteUnused:
         arguments: ['@doctrine.orm.default_entity_manager', '@united.cms.field_type_manager']
         tags:
-            - { name: 'console.command', command: 'united:storage:delete-unused' }
\ No newline at end of file
+            - { name: 'console.command', command: 'united:storage:delete-unused' }
+
+    # A service for S3 presigning from type
```

Comment:
Comment is a bit misleading. This is not a service FOR forms, this is a form-as-a-serivce.